README.md

```diff
@@ -21,7 +21,11 @@ The MSAL preview library for iOS and macOS gives your app the ability to begin u
 
 ```Swift
 do {
-    let application = try MSALPublicClientApplication.init(clientId: <your-client-id-here> authority: kAuthority)
+    // Create an instance of MSALPublicClientApplication with proper config
+    let authority = try MSALB2CAuthority.init(url: URL(string:kAuthority)!)

```

Comment:
nit: please remove init in the readme :)

Reply:
done, forgot to update it :P